tests/integration/_validator/testcases/test_validator.go

```diff
@@ -129,30 +130,46 @@ func (t *ValidatorTest) testInstallInteractive(ctx *test.TestContext) (tr *test.
 	return common.ExecCLI(interactiveCmd, buffer, t.log, false)
 }
 
-func (t *ValidatorTest) testInstallInteractiveCheck(ctx *test.TestContext) (tr *test.TestResult) {
-	t.log.Printf("Executing testInstallInteractiveCheck")
+func (t *ValidatorTest) testInstallInteractiveApplyExistingCluster(ctx *test.TestContext) (tr *test.TestResult) {
+	t.log.Printf("Executing testInstallInteractiveApply")
 
-	interactiveCmd, buffer := common.InitCmd([]string{"install", "-o", "--apply", "-l", "debug"})
+	kindClusterName = fmt.Sprintf("%s-%s", cfg.ValidatorKindClusterName, string_utils.RandStr(5))
```

Comment:
None of these kind cluster related changes are required. Please revert them and test with `DISABLE_KIND_CLUSTER_CHECK=true`.